app/controllers/controller.js

```diff
@@ -33,34 +33,50 @@
         if (template.length < 1)
             return fieldsetConfig.label;
 
-        var rgx = /{{(.*?)}}*/g;
+        var rgx = /{{([^)].*)}}/g;
```

Comment:
@kjac Any chance you can test these changes so they don't break your previous PR for allowing multiple functions?